cypress/e2e/chartgen.cy.js

```diff
@@ -0,0 +1,22 @@
+describe('Chart Generation Test', () => {
+  it('Generates chart correctly', () => {
+    //Visit the line page
+    cy.visit('/line.html');
+
+    //Enter chart title
+    cy.get('#chart-title-input').type('title')
+    //Enter x label and y label
+    cy.get('#x-label-input').type('x');
+    cy.get('#y-label-input').type('y');
+
+    //Enter x and y values
+    cy.get('[data-testid="x-input"]').type('7');
+    cy.get('[data-testid="y-input"]').type('5');
+
```

Comment:
The video calls out changing the chart color as well; could you please add an action for that just to be safe?